.circleci/config.yml

```diff
@@ -80,6 +80,45 @@ jobs:
       - store_artifacts:
           path: ui/cypress/screenshots
           destination: screenshots
+  selenium_accept:
+    docker:
+      - image: circleci/node:13.0.1-stretch-browsers
+      - image: quay.io/influxdb/influx:nightly
+        command: [ --e2e-testing=true ]
+    steps:
+      - checkout
+      - run:
+         name: Environment check
+         command: |
+           git --version
+           node --version && npm --version
+           docker --version
+           google-chrome --version && which google-chrome && chromedriver --version && which chromedriver
+      - run:
+         name: Checkout Tests
+         command: git clone --single-branch --branch 2.0-Alpha https://github.com/bonitoo-io/selenium-accept-infl2.git
+      - run:
+         name: Selenium tests
+         command: |
+           set +e
+           cd selenium-accept-infl2
+           npm install
+           npm test; TEST_RESULT=$?
+           sleep 30
```

Comment:
What is the purpose of `sleep`?

Reply:
Earlier in the week some data-testid's were changed, which lead to waits on elements to hang beyond the end of the test, which then lead to cucumber not writing out reports.  The sleep was added while troubleshooting the issue. I kept it as a kind of insurance, but have now removed it.  

I just added a line using curl to ensure the endpoint is up and can be used before proceeding with tests.